gravitee-node-vertx/src/test/java/io/gravitee/node/vertx/client/http/VertxHttpClientFactoryTest.java

```diff
@@ -0,0 +1,421 @@
+package io.gravitee.node.vertx.client.http;
+
+import static io.gravitee.node.vertx.client.http.VertxHttpClientFactory.HTTP_SSL_OPENSSL_CONFIGURATION;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertThrows;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.gravitee.node.api.configuration.Configuration;
+import io.gravitee.node.vertx.client.ssl.SslOptions;
+import io.gravitee.node.vertx.client.ssl.jks.JKSKeyStore;
+import io.gravitee.node.vertx.client.ssl.jks.JKSTrustStore;
+import io.gravitee.node.vertx.client.ssl.pem.PEMKeyStore;
+import io.gravitee.node.vertx.client.ssl.pem.PEMTrustStore;
+import io.gravitee.node.vertx.client.ssl.pkcs12.PKCS12KeyStore;
+import io.gravitee.node.vertx.client.ssl.pkcs12.PKCS12TrustStore;
+import io.vertx.rxjava3.core.Vertx;
+import io.vertx.rxjava3.core.http.HttpClient;
+import java.io.IOException;
+import java.util.Base64;
+import java.util.Objects;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+/**
+ * @author Yann TAVERNIER (yann.tavernier at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@ExtendWith(MockitoExtension.class)
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
```

Comment:
As you are not using _ in the test method name. This can be removed or change the method name to use snake case 🙈